app/upgrades/v0.0.11.go

```diff
@@ -0,0 +1,93 @@
+package upgrades
+
+import (
+	"context"
+
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	cosmosTypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/pokt-network/poktroll/app/keepers"
+	tokenomicstypes "github.com/pokt-network/poktroll/x/tokenomics/types"
+)
+
+// Upgrade_0_0_11 is the upgrade handler for v0.0.11 Alpha TestNet upgrade
+// Beta TestNet was launched with v0.0.11, so this upgrade is exclusively for Alpha TestNet.
+//   - Before: v0.0.10
+//   - After: v0.0.11
+var Upgrade_0_0_11 = Upgrade{
+	PlanName: "v0.0.11",
+	CreateUpgradeHandler: func(mm *module.Manager,
+		keepers *keepers.Keepers,
+		configurator module.Configurator,
+	) upgradetypes.UpgradeHandler {
+
+		// Adds new parameters using ignite's config.yml as a reference. Assuming we don't need any other parameters.
+		// https://github.com/pokt-network/poktroll/compare/v0.0.10...v0.0.11-rc
+		applyNewParameters := func(ctx context.Context) (err error) {
+			logger := cosmosTypes.UnwrapSDKContext(ctx).Logger()
+			logger.Info("Starting parameter updates for v0.0.11")
+
+			// Set num_suppliers_per_session to 15
+			// Validate with: `poktrolld q session params --node=https://testnet-validated-validator-rpc.poktroll.com/`
+			sessionParams := keepers.SessionKeeper.GetParams(ctx)
+			logger.Info("Current session params", "params", sessionParams)
+			sessionParams.NumSuppliersPerSession = uint64(15)
+			err = keepers.SessionKeeper.SetParams(ctx, sessionParams)
+			if err != nil {
+				logger.Error("Failed to set session params", "error", err)
+				return err
+			}
+			logger.Info("Successfully updated session params", "new_params", sessionParams)
+
+			// Set tokenomics params. The values are based on default values for LocalNet/Beta TestNet.
+			// Validate with: `poktrolld q tokenomics params --node=https://testnet-validated-validator-rpc.poktroll.com/`
+			tokenomicsParams := keepers.TokenomicsKeeper.GetParams(ctx)
```

Comment:
Here's the line that caused a panic: https://gist.github.com/okdas/50f417e3a86230e9e1aea85da841e1de

I need to investigate why I did not have this issue during local upgrade from v0.0.10 to v0.0.11. Will try to replicate using ONLY binaries provided in releases.

Could it be the before state is different on Alpha TestNet? If so, going to document this so we check for that in the future.

Reply:
> Here's the line that caused a panic: [gist.github.com/okdas/50f417e3a86230e9e1aea85da841e1de](https://gist.github.com/okdas/50f417e3a86230e9e1aea85da841e1de)

Why did this cause the panic?

Reply:
@Olshansk My first hypothesis it has happened due to the protobuf type change, but that should have been caught on the local test. For some reason, it did not. I'm going to be looking into that today.

I expect we will change an upgrade procedure to migrate each module individually, which can preserve the types between upgrades. There are some pros and cons associated with that. So far, I wanted the upgrade process to look simple, but it seems like additional complexity might worth it to avoid such issues in the future. 

Basically, I need to spend more time to provide a complete answer.

Reply:
> I need to spend more time to provide a complete answer.

This is REALLY REALLY REALLY REALLY important, so please dive as deep as necessary to the finest grain of detail.